Moto/E6Plus/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.moto.e6plus"
+        android:versionCode="1"
+        android:versionName="1.0">
+
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+                android:requiredSystemPropertyValue="+*pokerp"
```

Comment:
Is it tested? I'm surprised this work

Reply:
seriously? go then I think the users who used it deceived me 😭😭